File: sandbox/fabio/NewCecilPerfTest/NewCecilPerfTest/Data/Pilot.cs
using System;
namespace NewCecilPerfTest.Data
{
	public class Pilot
	{
		public string name;
		public int points;
		public Pilot (string name, int points)
		{
			this.name = name;
			this.points = points;
		}
		
		public override string ToString()
		{
			return "Pilot[" + name + ";"+points+"]";
		}
	}
}

File: reference/CodeExamples/javaDRS/src/com/db4odoc/drs/advanced/Pilot.java
package com.db4odoc.drs.advanced;

public class Pilot {
	private String name;
	int points;

    public Pilot(String name)  {
        this.name=name;
    }
    public Pilot(String name, int points)  {
        this.name=name;
        this.points = points;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getName()  {
        return name;
    }

    public int getPoints() {
        return points;
    }

    public void setPoints(int points) {
        this.points = points;
    }

    public String toString()  {
        return name;
    }
}